Build failure due to new version of staticcheck

Staticcheck added ST1021 which confirms the first word in a type comment matches the name of the type. This check should be disabled until the goa code generation templates are updated to conform with this validation.

https://github.com/dominikh/go-tools/commit/2c70f907df0394e2fc6c63b04fdfd5890f5a260d

Current lint target fails with:
```
make lint
LINTING CODE...
basic/gen/calc/service.go:15:1: comment on exported type Service should be of the form "Service ..." (with optional leading article) (ST1021)
cellar/gen/sommelier/service.go:17:1: comment on exported type Service should be of the form "Service ..." (with optional leading article) (ST1021)
cellar/gen/sommelier/service.go:83:1: comment on exported type NoCriteria should be of the form "NoCriteria ..." (with optional leading article) (ST1021)
cellar/gen/sommelier/service.go:86:1: comment on exported type NoMatch should be of the form "NoMatch ..." (with optional leading article) (ST1021)
cellar/gen/storage/service.go:17:1: comment on exported type Service should be of the form "Service ..." (with optional leading article) (ST1021)
...
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,7 +4,6 @@ go:
 - "1.12.x"
 cache:
   directories:
-    - $HOME/.cache/go-build
     - $HOME/gopath/pkg
     - $HOME/gopath/src/bitbucket.org
     - $HOME/gopath/src/cloud.google.com

--- a/Makefile
+++ b/Makefile
@@ -75,7 +75,7 @@ lint:
 	@if [ "`golint ./... | grep -vf .golint_exclude | tee /dev/stderr`" ]; then \
 		echo "^ - Lint errors!" && echo && exit 1; \
 	fi
-	@if [ "`staticcheck -checks all,-ST1000,-ST1001 ./... | grep -v ".pb.go" | tee /dev/stderr`" ]; then \
+	@if [ "`staticcheck -checks all,-ST1000,-ST1001,-ST1021 ./... | grep -v ".pb.go" | tee /dev/stderr`" ]; then \
 		echo "^ - staticcheck errors!" && echo && exit 1; \
 	fi
 

